api/src/main/java/com/graphhopper/routing/util/CustomModel.java

```diff
@@ -126,14 +126,13 @@ public double getHeadingPenalty() {
 
     @Override
     public String toString() {
-        return "CustomModel{" +
-                "distanceInfluence=" + distanceInfluence +
-                ", speedFactor=" + speedFactorMap +
-                ", maxSpeed=" + maxSpeedMap +
-                ", maxSpeedFallback=" + maxSpeedFallback +
-                ", priorityMap=" + priorityMap +
-                ", #areas=" + areas.size() +
-                '}';
+        return createContentString();
+    }
+
+    private String createContentString() {
```

Comment:
Should we not use createContentString also in getVersion() (instead of toString()) now that we have it?

Reply:
Ah sorry its in another file. But maybe we can do the same in `Profile.java` (add `createContentString` and use it from `Profile#getVersion()` and `Profile#getVersion()`)?

Reply:
Yes!